x-pack/plugins/apm/public/components/app/diagnostics/apm_documents_tab.tsx

```diff
@@ -123,13 +127,32 @@ export function DiagnosticsApmDocuments() {
       {isImported && diagnosticsBundle ? (
         <>
           <EuiBadge>
-            From: {new Date(diagnosticsBundle.params.start).toISOString()}
+            {i18n.translate(
+              'xpack.apm.diagnosticsApmDocuments.from:BadgeLabel',
+              { defaultMessage: 'From:' }
+            )}
+            {new Date(diagnosticsBundle.params.start).toISOString()}
           </EuiBadge>
           <EuiBadge>
-            To: {new Date(diagnosticsBundle.params.end).toISOString()}
+            <FormattedMessage
+              id="xpack.apm.diagnosticsApmDocuments.to:BadgeLabel"
+              defaultMessage="To:"
+            />
+            {new Date(diagnosticsBundle.params.end).toISOString()}
```

Comment:
No need to use `FormattedMessage` here right? 
```suggestion
           {i18n.translate(
              'xpack.apm.diagnosticsApmDocuments.to:BadgeLabel',
              { defaultMessage: 'To: {date}', 
              values: {date: new Date(diagnosticsBundle.params.end).toISOString()} }
            )}
```